tgui/packages/tgui/interfaces/MafiaPanel.js

```diff
@@ -0,0 +1,161 @@
+import { FlexItem } from '../components/Flex';
+
+export const MafiaPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    players,
+    actions,
+    phase,
+    roleinfo,
+    role_theme,
+    admin_controls,
+    judgement_phase,
+    timeleft,
+    all_roles } = data;
+  return (
+    <Window
+      theme={role_theme}
+      resizable>
+      <Window.Content>
+        <Section title={phase}>
+          {!!roleinfo && (
+            <Table>
+              <Table.Row>
+                <Table.Cell>
+                  <TimeDisplay auto="down" value={timeleft} />
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row>
+                <Table.Cell bold>
+                  You are a {roleinfo.role}
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row bold>
+                <Table.Cell>
+                  {roleinfo.desc}
+                </Table.Cell>
+              </Table.Row>
+            </Table>
+          )}
+        </Section>
+        <Flex>
+          {!!actions && actions.map(action => {
+            return (
+              <Flex.Item key={action}>
+                <Button
+                  onClick={() => act("mf_action", { atype: action })}>
+                  {action}
+                </Button>
+              </Flex.Item>);
+          })}
+        </Flex>
+        {!!admin_controls && (
+          <Section
+            title="ADMIN CONTROLS"
+            backgroundColor="red">
+            THESE ARE DEBUG, THEY WILL BREAK THE GAME, DO NOT TOUCH <br />
+            Also because an admin did it: do not gib
+            anyone! It will runtime the game to death! <br />
+            <Button
+              icon="arrow-right"
+              onClick={() => act("next_phase")}>Next Phase
+            </Button>
+            <Button
+              icon="home"
+              onClick={() => act("players_home")}>Send All Players Home
+            </Button>
+            <Button
+              icon="radiation"
+              onClick={() => act("new_game")}>New Game
+            </Button>
+          </Section>
+        )}
+        <Section title="Players">
+          <LabeledList>
+            {!!players && players.map(player => { return (
+              <LabeledList.Item
+                className="candystripe"
+                key={player.ref}
+                label={player.name}>
+                {!player.alive && (<Box color="red">DEAD</Box>)}
+                {player.votes !== undefined && !!player.alive
+                && (<Fragment>Votes : {player.votes} </Fragment>)}
+                {
+                  !!player.actions && player.actions.map(action => {
+                    return (
+                      <Button
+                        key={action}
+                        onClick={
+                        // eslint-disable-next-line indent
+      () => act("mf_targ_action", { atype: action, target: player.ref })
+                        }>
```

Comment:
Please don't silence this obviously bad indent. Split act parameters over multiple lines.
```suggestion
                        onClick={() => act('mf_targ_action', {
                          atype: action,
                          target: player.ref,
                        })}>
```